weblate/accounts/notifications.py

```diff
@@ -529,6 +529,29 @@ class NewSuggestionNotificaton(Notification):
     required_attr = "suggestion"
 
 
+@register_notification
+class LanguageTranslatedNotificaton(Notification):
+    actions = (Change.ACTION_COMPLETE,)
+    verbose = pgettext_lazy("Notification name", "Language was translated")
+    template_name = "translated_language"
+
+    def get_context(
+        self, change=None, subscription=None, extracontext=None, changes=None
+    ):
+        context = super().get_context(change, subscription, extracontext, changes)
+        if change:
+            context["language"] = Language.objects.get(code=change.details["language"])
+        return context
```

Comment:
`get_context` should not be needed.

```suggestion
    required_attr = "translation"
```